src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/session/GetSessionsTest.kt

```diff
@@ -59,6 +59,40 @@ class GetSessionsTest(@Autowired private val objectMapper: ObjectMapper) : Integ
     assertSession(visitSessionResults[0], nextAllowedDay, sessionTemplate)
   }
 
+  @Test
+  fun `visit sessions are returned for enhanced prisoner a prison for a single schedule`() {
```

Comment:
Should we add another test to ensure enhanced sessions are not returned for non enhanced prisoners?

Reply:
will do this when I merge your code with it I think!